pkg/volume/metrics_du_test.go

```diff
@@ -23,6 +23,8 @@ import (
 	"testing"
 )
 
+const expectedBlockSize = 4096
```

Comment:
while VERY usually correct, not all blocks are 4k.   should the test get the block size from stat info and just accept either 0 or 1 blocks in size?  maybe that doesn't help either...